### DISCUSSION TITLE
[Question] Tabs in view

### DISCUSSION THREAD

**Discussion by LuciferEwah** [flet-dev/flet#1986] [Q&A]
### Question

I'm encountering an issue with the flet library in my Python code. I'm trying to create tabs with views using ft.View, but I'm getting an "Exception: Unknown control type: view" error. I've checked my code, and it seems like the ft.View class should be valid. This is for a larger project, but I wanted to start with an isolated test. Could someone please help me understand why I'm getting this error and how to resolve it?

### Code sample

```python
import flet as ft
import logging

logging.basicConfig(level=logging.DEBUG)


def main(page: ft.Page):

    # Crear instancias de las vistas que deseas agregar a las pestañas
    view1 = ft.View('/view1', controls=[ft.Text("Contenido de la Vista 1")])
    view2 = ft.View('/view2', controls=[ft.Text("Contenido de la Vista 2")])
    view3 = ft.View('/view3', controls=[ft.Text("Contenido de la Vista 3")])

    # Crear las pestañas y agregar las vistas como contenido
    t = ft.Tabs(
        selected_index=0,  # Índice de la pestaña seleccionada inicialmente
        animation_duration=300,
        tabs=[
            ft.Tab(
                text="Tab 1",
                content=view1,
            ),
            ft.Tab(
                tab_content=ft.Icon(ft.icons.SEARCH),
                content=view2,
            ),
            ft.Tab(
                text="Tab 3",
                icon=ft.icons.SETTINGS,
                content=view3,
            ),
        ],
        expand=1,
    )

    page.add(t)  # Agregar las pestañas a la página principal

if __name__ == '__main__':
    ft.app(target=main)
```

### Error message

```bash
DEBUG:flet_runtime:_on_message: {"action":"pageEventFromWeb","payload":{"eventTarget":"page","eventName":"error","eventData":"Exception: Unknown control type: view"}} 
DEBUG:flet_core:page.on_event: page error Exception: Unknown control type: view
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
`View` is actually not meant to be used as such. It is mainly for routing and navigation purposes.
See [this](https://flet.dev/docs/guides/python/navigation-and-routing) for more.

As `content` of your `Tab`, you should check [other controls](https://flet.dev/docs/controls) such as Row, Column etc.

**Comment by LuciferEwah** [comment]
thanks you
